src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,41 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+
+public class LoginPage extends BasePage{
+    public LoginPage(WebDriver givenDriver) {
+
+        super(givenDriver);
+    }
+    //WebElement
+        By emailField = By.cssSelector("[type=\"email\"]");
+        By passwordField = By.cssSelector("[type=\"password\"]");
+        By submitBtn = By.cssSelector("[type=\"submit\"]");
+        By avatar = By.cssSelector("[class=\"avatar\"]");
+        By koelsym = By.cssSelector("[class=\"logo\"]");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```